packages/sdk-js/src/models/node/contractWrite.ts

```diff
@@ -3,7 +3,7 @@ import Node from "./interface";
 import * as avs_pb from "@/grpc_codegen/avs_pb";
 import { NodeType } from "@avaprotocol/types";
 
-// Required props for constructor: id, name, type and data: { contractAddress, callData, contractAbi }
+// Required props for constructor: id, name, type and data: { config: { contractAddress, callData, contractAbi } }
 export type ContractWriteNodeData = avs_pb.ContractWriteNode.AsObject;
```

Comment:
The ContractWriteNodeData is meant to be assigend to avs_pb.ContractWriteNode.getConfig().AsObject. This way, we don’t have to embed the config into data, but use data as the config object. Could you try to make this change to contractWrite.ts? If it works, this format should apply to other Node.